### PR TITLE
Scraper for Western Australia

### PR DESCRIPTION
**Pull request by MariusKroon**
First attempt at creating a scraper for Western Australia

### CODE REVIEW

File: parsers/AU-WA.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        production.drop('Distillate', inplace=True)
+        production.ix['Other'] = production.ix['Landfill Gas']
+        production.drop('Other', inplace=True)
+                
+        rptData = production.to_dict()
+        data = {
+            'countryCode': country_code,
+            'production': rptData['production'],
+            'capacity': rptData['capacity'],
+            'datetime': str(dumpDate),
```
Comment (corradio) [inline parsers/AU-WA.py]: This actually need to be a python datetime object, not a string.

File: parsers/AU-WA.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            'storage': {},
+            'source': 'http://wa.aemo.com.au/',
```
Comment (corradio) [inline parsers/AU-WA.py]: I tend to only post the domain name for consistency. Can you change to `wa.aemo.com.au`?

File: parsers/AU-WA.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import arrow
```
Comment (corradio) [inline parsers/AU-WA.py]: File name needs to be with an underscore instead of dash (`AU_WA.py`).

**Comment by corradio** [comment]
Fixes https://github.com/tmrowco/electricitymap/issues/564

File: parsers/AU-WA.py (hunk 2 above)
Comment (MariusKroon) [inline parsers/AU-WA.py]: Fixed in next version

File: parsers/AU-WA.py (hunk 1 above)
Comment (MariusKroon) [inline parsers/AU-WA.py]: Fixed in next version

File: parsers/AU-WA.py (hunk 3 above)
Comment (MariusKroon) [inline parsers/AU-WA.py]: fixed in next version

Comment (corradio) [inline parsers/AU-WA.py]: This file should probably be deleted right?

File: README.md [hunk 4]
```diff
@@ -41,6 +41,7 @@ Each country has a CO2 mass flow that depends on neighboring countries. In order
 Real-time electricity data is obtained using [parsers](https://github.com/corradio/electricitymap/tree/master/parsers)
 
 - Australia: [AREMI National Map](http://services.aremi.nationalmap.gov.au/aemo/v3/csv/all)
+- Western Australie: [AEMO] (http://wa.aemo.com.au/aemo/data/wa/infographic/facility-intervals-last96.csv)
```
Comment (corradio) [inline README.md]: For consistency with the app, I believe we should write `Australia (Western)` or add it as a subitem.

File: web/app/configs/lang.json [hunk 5]
```diff
@@ -664,6 +664,7 @@
 
         "zoneShortName": {
             "AL": "Albanië",
+
```
Comment (corradio) [inline web/app/configs/lang.json]: why this newline?

File: web/app/configs/lang.json [hunk 6]
```diff
@@ -694,6 +695,7 @@
             "MT": "Malta",
             "NL": "Nederland",
             "NO": "Noorwegen",
+			"PE": "Peru",
```
Comment (corradio) [inline web/app/configs/lang.json]: Seems like indentation is wrong.

File: web/app/configs/lang.json [hunk 7]
```diff
@@ -703,6 +705,7 @@
             "SI": "Slovenië",
             "SK": "Slowakije",
             "TR": "Turkije",
+			"TW": "Taiwan",
```
Comment (corradio) [inline web/app/configs/lang.json]: Seems like indentation is wrong.

File: parsers/AU-WA.py (hunk 3 above)
Comment (MariusKroon) [inline parsers/AU-WA.py]: Correct - I renamed it to AU_WA. Drop it like it's hot.

File: web/app/configs/lang.json (hunk 7 above)
Comment (MariusKroon) [inline web/app/configs/lang.json]: I have to check my editor why it keeps doing that.

File: parsers/AU-WA.py (hunk 3 above)
Comment (corradio) [inline parsers/AU-WA.py]: 🔥 🤣

File: parsers/config.py [hunk 8]
```diff
@@ -94,6 +94,7 @@
     'AUS-SA': AU.fetch_production,
     'AUS-TAS': AU.fetch_production,
     'AUS-VIC': AU.fetch_production,
+    'AUS-WA': AU_WA.fetch_production,
```
Comment (corradio) [inline parsers/config.py]: Still requires the import (check the top of the file)

**Comment by MariusKroon** [comment]
I'll get there :) 
It's still a big learning curve for me, the Peruvian scraper helped a great deal as example.

**Comment by corradio** [comment]
Awesome! Don't hesitate to ask on Slack if you need help :)

**Comment by corradio** [comment]
Seems to work 🎉 
image

Feel free to announce it on Twitter/Facebook and then we can RT/share!

**Pull request merged**